runway/hooks/staticsite/auth_at_edge/lambda_config.py

```diff
@@ -0,0 +1,144 @@
+"""CFNgin prehook responsible for creation of Lambda@EDGE functions."""
+
+import logging
+import os
+import tempfile
+# pylint: disable=import-error, no-name-in-module
+from distutils.dir_util import copy_tree
+from tempfile import mkstemp
+# pylint: disable=unused-import
```

Comment:
this should be in-line so it does not apply to the entire file, only a single line